QKSMS/src/main/java/com/moez/QKSMS/ui/conversationlist/ConversationListFragment.java

```diff
@@ -195,6 +196,8 @@ public void onItemLongClick(final Conversation conversation, View view) {
 
         dialog.addMenuItem(R.string.menu_delete_conversation, MENU_DELETE_CONVERSATION);
 
+        dialog.addMenuItem(R.string.delete_failed, MENU_DELETE_FAILED);
```

Comment:
Don't need to add this option if the conversation doesn't even have any failed messages